lib/vdsm/virt/vm.py

```diff
@@ -5149,11 +5143,18 @@ def onIOError(self, blockDevAlias, err, action):
             self._pause_time = vdsm.common.time.monotonic_time()
             self._setGuestCpuRunning(False, flow='IOError')
             self._logGuestCpuStatus('onIOError')
-            if reason == 'ENOSPC':
-                if not self.monitor_volumes():
-                    self.log.info("No volumes were extended")
 
-            self._send_ioerror_status_event(reason, blockDevAlias)
+            try:
+                drive = vmdevices.lookup.device_by_alias(
+                    self._devices[hwclass.DISK][:], blockDevAlias)
+            except LookupError:
+                drive = None
+                self.log.warning('unknown disk alias: %s', blockDevAlias)
```

Comment:
Start with uppercase?

Reply:
I just moved existing log. In general we start log with capital letter but some logs
are not consistent. Making logs consistent is unrelated task that should be done later.